travis: golint-ci fails build due to "unused" provider sets in package main

While testing a PR, I noticed that Travis flagged a Wire provider set as an unused variable ([source](https://travis-ci.com/google/go-cloud/builds/75304110#L661)):

```
samples/guestbook/main.go:269:5: `appSet` is unused (varcheck)
var appSet = wire.NewSet(
    ^
```

This is technically correct, but not an actual issue, and it did cause the build to fail, thus blocking submission.  Based on recommendation from @cflewis, I'm disabling `varcheck` as a temporary workaround, then keeping this open for solving the underlying issue.

--- a/blob/gcsblob/gcsblob.go
+++ b/blob/gcsblob/gcsblob.go
@@ -105,8 +105,6 @@ func (b *bucket) Delete(ctx context.Context, key string) error {
 	return err
 }
 
-const namingRuleURL = "https://cloud.google.com/storage/docs/naming"
-
 func bufferSize(size int) int {
 	if size == 0 {
 		return googleapi.DefaultUploadChunkSize

--- a/internal/testing/checkpr.sh
+++ b/internal/testing/checkpr.sh
@@ -65,5 +65,5 @@ golangci-lint run \
   --disable=deadcode \
   --disable=typecheck \
   --disable=unused \
-  --disable=varcheck \
+  --build-tags wireinject \
   "${lintdirs[@]}" || exit 1

--- a/internal/testing/checkpr.sh
+++ b/internal/testing/checkpr.sh
@@ -65,5 +65,5 @@ golangci-lint run \
   --disable=deadcode \
   --disable=typecheck \
   --disable=unused \
-  --build-tags wireinject \
+  --build-tags=wireinject \
   "${lintdirs[@]}" || exit 1

